app/src/main/java/edu/uc/hickmadc/customnotifications/adapters/GroupsAdapter.kt

```diff
@@ -48,21 +41,18 @@ class GroupsAdapter(private val dataSet: ArrayList<Group>) :
 //            }
         }
 
-        fun bind(item: Group) {
+        internal fun bind(item: Group) {
             binding.apply {
                 group = item
                 executePendingBindings()
             }
         }
     }
 
-    /**
-     * Handles the comparison between groups.
-     */
     private class GroupDiffCallback : DiffUtil.ItemCallback<Group>() {
 
         override fun areItemsTheSame(oldItem: Group, newItem: Group): Boolean {
-            return oldItem.groupId == newItem.groupId
+            return oldItem.group_id == newItem.group_id
```

Comment:
Was better before.  groupId is proper lowerCamelCase naming convetion; group_id is not.